packages/fxa-auth-server/test/oauth/routes/authorization.js

```diff
@@ -92,6 +92,22 @@ describe('/authorization POST', function () {
       );
     });
 
+    // https://github.com/mozilla/fxa/issues/5660
+    it('accepts TTL larger than the maximum supported', () => {
+      const ONE_YEAR_IN_SECONDS = 31536000;
+      Joi.assert(
+        {
+          client_id: CLIENT_ID,
+          assertion: BASE64URL_STRING,
+          scope: 'bar',
+          state: 'foo',
+          response_type: 'token',
+          ttl: ONE_YEAR_IN_SECONDS,
+        },
+        validation
+      );
+    });
+
```

Comment:
Testing nit: I think it would also be worth testing that the route actually returns the desired smaller TTL in this case.